### DISCUSSION TITLE
Add Stable Diffusion 2.0 models

### DISCUSSION THREAD

**Discussion by mrauhu** [AbdBarho/stable-diffusion-webui-docker#240] [Ideas]
Hello, @AbdBarho.

[Stable Diffusion 2.0](https://stability.ai/blog/stable-diffusion-v2-release) is released.

## Models

* 768x768
https://huggingface.co/stabilityai/stable-diffusion-2
  * https://huggingface.co/stabilityai/stable-diffusion-2/blob/main/768-v-ema.ckpt
* 512x512
https://huggingface.co/stabilityai/stable-diffusion-2-base
   * https://huggingface.co/stabilityai/stable-diffusion-2-base/resolve/main/512-base-ema.ckpt
* Depth-to-Image
https://huggingface.co/stabilityai/stable-diffusion-2-depth
  * https://huggingface.co/stabilityai/stable-diffusion-2-depth/blob/main/512-depth-ema.ckpt
* Inpainting 
https://huggingface.co/stabilityai/stable-diffusion-2-inpainting
  * https://huggingface.co/stabilityai/stable-diffusion-2-inpainting/blob/main/512-inpainting-ema.ckpt
* Super-resolution Upscaler Diffusion
https://huggingface.co/stabilityai/stable-diffusion-x4-upscaler
  * https://huggingface.co/stabilityai/stable-diffusion-x4-upscaler/blob/main/x4-upscaler-ema.ckpt

### See also

* https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/5011

Can you add new models to the `download` service?

Best wishes,
Sergey.

**Comment by AbdBarho** [comment]
@mrauhu cool!

It is not backwards compatible so we are going to have to wait for the UIs to get updated.

Exciting stuff!

**Reply by ryanbrackney** [reply]
Did this ever happen? The download docker compose still only brings in 1.5.

**Reply by AbdBarho** [reply]
@ryanbrackney https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/244

It is still in progress, I don't know why but the generated images are incoherent:

![00410-2198486075-cat](https://user-images.githubusercontent.com/24505302/204880289-7bd33e54-095d-4c83-8c68-37f127617aa1.png)
